File: src/main/java/com/fish/entity/vo/ColumnVO.java
package com.fish.entity.vo;

import java.io.Serializable;
import java.util.ArrayList;

public class ColumnVO implements Serializable {
    private Long columnId;
    private String columnName;
    private String columnDescribe;
    private Boolean enabled;
    ArrayList<TopicVO> topicVOS;
}

File: src/main/kotlin/com/fish/keXieOpenJudge/entity/vo/ColumnVO.kt
package com.fish.keXieOpenJudge.entity.vo

import java.io.Serializable

data class ColumnVO(
    var columnId: Long? = null,
    var columnName: String? = null,
    var columnDescribe: String? = null,
    var enabled: Boolean? = null,
    var topicVOS: ArrayList<TopicVO>? = null
) : Serializable
